CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 * Prioritize integration of local changes over remote changes - shorten the time users may have to wait when committing local changes. Stop storing downloaded changesets in history. (Core upgrade)
 * Greatly improve the performance of sorting or distincting a Dictionary's keys or values. The most expensive operation is now performed O(log N) rather than O(N log N) times, and large Dictionaries can see upwards of 99% reduction in time to sort. (Core upgrade)
 * Seamlessly handle migrating an App Services application deployment model. (Core upgrade)
+* Slightly improve performance of `Realm.RemoveAll()` which removes all objects from an open Realm database. (Issue [#2233](https://github.com/realm/realm-dotnet/issues/2194))
```

Comment:
let me know if I should word this differently

Reply:
I think we should specify that we also changed the behaviour (and eventually change the text for the docs too)